File: app/src/main/java/com/friendlypos/distribucion/modelo/InventarioResponse.java
package com.friendlypos.distribucion.modelo;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by DelvoM on 31/10/2017.
 */

public class InventarioResponse {

    private boolean result;
    private String code;
    private String message;
    @SerializedName("inventory")
    private List<Inventario> inventarios;

    public boolean isResult() {
        return result;
    }

    public void setResult(boolean result) {
        this.result = result;
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public List<Inventario> getInventarios() {
        return inventarios;
    }

    public void setInventarios(List<Inventario> inventarios) {
        this.inventarios = inventarios;
    }
}

File: app/src/main/java/com/friendlypos/distribucion/modelo/InventarioResponse.kt
package com.friendlysystemgroup.friendlypos.distribucion.modelo

import com.google.gson.annotations.SerializedName

/**
 * Created by DelvoM on 31/10/2017.
 */
class InventarioResponse {
    var isResult: Boolean = false
    var code: String? = null
    var message: String? = null

    @JvmField
    @SerializedName("inventory")
    var inventarios: List<Inventario>? = null
}
